README.md

```diff
@@ -1,12 +1,11 @@
-open-orchestra-media-bundle
+open-orchestra-workflow-bundle
```

Comment:
Wouldn't it be more simple to call it workflow-bundle, to integrate new future workflow notions & features not related to fonctions ?